### DISCUSSION TITLE
many free channels

### DISCUSSION THREAD

**Discussion by pendy99** [iptv-org/community#82] [Show and tell]
I've found a great site with a ton of open source free channels that update automatically.
Please add these channels to your list.

https://epg.pm/test_channel_page.html

**Comment by BellezaEmporium** [comment]
I think most of the channels we don't have comes from this host : http://hls-ott-zhibo.wasu.tv/live/.../index.m3u8

**Comment by vijaibishnoi** [comment]
[Free World Iptv Channels Playlist ](https://www.techstag.in/2022/07/iptv-m3u8-playlist-links.html)
